File: 0650-2-keys-keyboard/0650-2-keys-keyboard.py
class Solution:
    def minSteps(self, n: int) -> int:
        if n == 1:
            return 0
        ans = 0
        while n > 1:
            for i in range(2, 1000):
                if n % i == 0:
                    ans += i
                    break
            n //= i
        return ans       

File: 0650-2-keys-keyboard/0650-2-keys-keyboard.cpp
class Solution {
public:
    int minSteps(int n) {
        if (n == 1) return 0;
        int ans = 0;
        while (n > 1) {
            for (int i = 2; i < 1000; i++) {
                if (n % i == 0) {
                    ans += i;
                    n /= i;
                    break;
                }
            }

        }

        return ans;
    }
};